### PR TITLE
[WIP] Allow super-admin to update invitation

### PR DESCRIPTION
**Pull request by DIOHz0r**
Hello,

This changes is for allow the super admin to update invitations. 

This fixed issue #84 

Regards.

### CODE REVIEW

File: install/installer.class.php [hunk 1]
```diff
@@ -230,7 +230,7 @@ protected function createFirstAccess() {
 
       $newRights = [
          PluginFlyvemdmProfile::$rightname         => PluginFlyvemdmProfile::RIGHT_FLYVEMDM_USE,
-         PluginFlyvemdmInvitation::$rightname      => CREATE | READ | DELETE | PURGE,
+         PluginFlyvemdmInvitation::$rightname      => CREATE | READ | UPDATE | DELETE | PURGE,
```
Comment (btry) [inline install/installer.class.php]: use ALLSTANDARDRIGHT whics is a shortcut for CREATE | READ | UPDATE | DELETE | PURGE

File: install/installer.class.php [hunk 2]
```diff
@@ -247,7 +247,12 @@ protected function createFirstAccess() {
          PluginFlyvemdmTaskstatus::$rightname      => READ,
       ];
 
-      $profileRight->updateProfileRights($_SESSION['glpiactiveprofile']['id'], $newRights);
+      $profiles_id = $_SESSION['glpiactiveprofile']['id'];
+      $profileRight->updateProfileRights($profiles_id, $newRights);
+      if($profiles_id != 4){
```
Comment (btry) [inline install/installer.class.php]: don't hard code a ID of profile. Consider the variable $profiles_id is the ID of the super admin profile. Thus the if() {} is useless,

File: install/upgrade/update_to_dev.php [hunk 3]
```diff
@@ -41,14 +41,19 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
...
+   $currentRights = ProfileRight::getProfileRights($profiles_id);
    $newRights = array_merge($currentRights, [
       PluginFlyvemdmInvitation::$rightname      => CREATE | READ | UPDATE | DELETE | PURGE,
       PluginFlyvemdmInvitationlog::$rightname   => READ,
       PluginFlyvemdmGeolocation::$rightname     => ALLSTANDARDRIGHT | READNOTE | UPDATENOTE,
       PluginFlyvemdmTask::$rightname            => READ,
    ]);
-   $profileRight->updateProfileRights($_SESSION['glpiactiveprofile']['id'], $newRights);
+   $profileRight->updateProfileRights($profiles_id, $newRights);
+   if($profiles_id != 4){
```
Comment (btry) [inline install/upgrade/update_to_dev.php]: same as above: use $profiles_id and consider it is the super admin profile.

**Comment by ajsb85** [comment]
The commit message type bug does not exist in our presets:
`bug(install):` use fix please.

**Pull request merged**